src/types/index.ts

```diff
@@ -658,3 +659,10 @@ export interface IWalletInfo {
   origin: any;
   type: any;
 }
+
+export type SupportedLanguage = keyof typeof SUPPORTED_LANGUAGES;
+
+export interface ILanguageConfig {
+  name: string;
+  getMessages: () => Promise<any>;
+}
```

Comment:
As far as I see this interface is used only in one file so it's not a global thing. I suggest to define it in the file that uses it.